.stylelintrc

```diff
@@ -3,6 +3,7 @@
     "stylelint-config-standard"
   ],
   "rules": {
+    "alpha-value-notation": null,
```

Comment:
stylelint-config-standard doesn't like alpha values of `0.5` anymore. It wants `50%` for some reason.

Figured it doesn't really matter to us, and turned off the rule.